config/newrelic.yml

```diff
@@ -0,0 +1,753 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python, Node, and Go applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+
+# Generated <%= Time.now.strftime('%B %d, %Y') %><%= ", for version #{@agent_version}" if @agent_version %>
+#<%= "\n# #{generated_for_user}\n#" if generated_for_user %>
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: 9942120fb257ae099f776017cd6926a760e6NRAL
```

Comment:
Should this be pulled from an ENV? I think the worst case is someone submitting bogus data to NR, and since we have a free account even if our resources get used up it wouldn't be the worst thing ever, so maybe fine for now.

Reply:
Good catch. Updated this.